receptor_satellite/response_queue.py

```diff
@@ -1,47 +1,26 @@
-import asyncio
-
-
-class ResponseQueue(asyncio.Queue):
-
-    def __init__(self, *args, **kwargs):
-        self.done = False
-        super().__init__(*args, **kwargs)
-
-    def __aiter__(self):
-        return self
-
-    async def __anext__(self):
-        while True:
-            try:
-                return self.get_nowait()
-            except asyncio.QueueEmpty:
-                if self.done:
-                    raise StopAsyncIteration
-            await asyncio.sleep(0.5)
+class ResponseQueue:
+    def __init__(self, queue):
+        self.queue = queue
 
     def ack(self, playbook_run_id):
-        payload = {
-            'type': 'playbook_run_ack',
-            'playbook_run_id': playbook_run_id
-        }
-        return self.put(payload)
+        payload = {"type": "playbook_run_ack", "playbook_run_id": playbook_run_id}
+        return self.queue.put(payload)
 
     def playbook_run_update(self, host, playbook_run_id, output, sequence):
         payload = {
-            'type': 'playbook_run_update',
-            'playbook_run_id': playbook_run_id,
-            'sequence': sequence,
-            'host': host,
-            'console': output
+            "type": "playbook_run_update",
+            "playbook_run_id": playbook_run_id,
+            "sequence": sequence,
+            "host": host,
+            "console": output,
         }
-        return self.put(payload)
+        return self.queue.put(payload)
```

Comment:
Whereas `asyncio.Queue.put` returns a coroutine, `queue.Queue.put` returns nothing - you may wish to consider removing the `return` so as not to confuse.

Reply:
Thanks, fixed